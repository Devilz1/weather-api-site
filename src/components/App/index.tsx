import React from "react";
import elf from 'img/elf.jpg';

import s from './style.scss';

const App = () => (
    <div>
        <h1>SITE SITE WWWWW</h1>
        <main>
            <div className={s.zvaal}>1</div>
            <div className={s.wow}>2</div>
            <div className={s.elf}><img src={elf} alt="Elf"/></div>
        </main>
    </div>
);

export default App;

