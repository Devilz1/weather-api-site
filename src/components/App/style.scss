@import "css/style";

:local {
  .rootClass {
    border-bottom: 4px solid black;
  }

  .zvaal {
    color: green;
    background-image: url("./../../img/zvaal.png");
    background-size: 100%;
    width: 100%;
    height: 150px;
    transition: 0.3s ease;

    &:hover {
      background-position: center;
    }
  }

  .wow {
    color: yellowgreen;
    background-image: url("./../../img/wow.svg");
    background-repeat: no-repeat;
    background-position-x: 50%;
    background-size: contain;
    height: 150px;
    width: 100%;
    display: flex;
    flex-direction: column;
    justify-content: center;
    transition: 0.9s ease;

    &:hover {
      filter: sepia(100%);
    }
  }

  .elf {
    color: aqua;
    background-image: url("./../../img/elf.jpg");
    background-repeat: no-repeat;
    background-size: 100%;
    height: 150px;
    width: 100%;
    display: flex;
    flex-direction: column;
    justify-content: center;
  }
}
