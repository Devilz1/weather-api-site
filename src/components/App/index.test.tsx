import React from 'react';
import renderer from 'react-test-renderer';
import App from './index';

describe('render component', () => {
    it('should correct render', () => {
        const wrapper = renderer.create(<App />).toJSON();
        expect(wrapper).toMatchSnapshot();
    });
});
