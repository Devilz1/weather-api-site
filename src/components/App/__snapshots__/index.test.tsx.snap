// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render component should correct render 1`] = `
<div>
  <h1>
    SITE SITE WWWWW
  </h1>
  <main>
    <div
      className="zvaal"
    >
      1
    </div>
    <div
      className="wow"
    >
      2
    </div>
    <div
      className="elf"
    >
      <img
        alt="Elf"
        src="test-file-stub"
      />
    </div>
  </main>
</div>
`;
